ibis/backends/tests/test_temporal.py

```diff
@@ -475,13 +508,13 @@ def test_to_timestamp(backend, con, unit):
 @pytest.mark.parametrize(
     ('date', 'expected_index', 'expected_day'),
     [
-        ('2017-01-01', 6, 'Sunday'),
-        ('2017-01-02', 0, 'Monday'),
-        ('2017-01-03', 1, 'Tuesday'),
-        ('2017-01-04', 2, 'Wednesday'),
-        ('2017-01-05', 3, 'Thursday'),
-        ('2017-01-06', 4, 'Friday'),
-        ('2017-01-07', 5, 'Saturday'),
+        param('2017-01-01', 6, 'Sunday', id="sunday"),
+        param('2017-01-02', 0, 'Monday', id="monday"),
+        param('2017-01-03', 1, 'Tuesday', id="tuesday"),
+        param('2017-01-04', 2, 'Wednesday', id="wednesday"),
+        param('2017-01-05', 3, 'Thursday', id="thursday"),
+        param('2017-01-06', 4, 'Friday', id="friday"),
+        param('2017-01-07', 5, 'Saturday', id="saturday"),
     ],
 )
```

Comment:
You can use the `ids` keyword for parametrize alternatively:

```suggestion
    ids=lambda tpl: tpl[2].lower()
)
```

Reply:
This is called for every argument unfortunately so I don't think there's much deduplication to be had here.